modules/doc/content/python/testers/Exodiff.md

```diff
@@ -42,17 +42,13 @@ ${MOOSE_DIR}/framework/contrib/exodiff/exodiff -summary moose_simulation_out.e
 
 This generates a summary of the simulation which can be edited to make the
 custom comparison file, where the tolerances for each variable may be set individually.
-In the example shown below, the relative tolerance criterion for the `z_field` variable
-is modified from the global value of `5.5e-6` to `10`. This essentially disables
-comparison for that field.
+In the example below, the tolerances are adjusted for local comparisons of field variables,
+both nodal and elemental. The relative error criteria is set to `1e-6` and the absolute
+criteria is disabled by setting it to `0`. These modifications apply to variable `u` and `distance`,
```

Comment:
By "absolute criteria" do you mean the floor parameter? This effectively sets the absolute zero parameter for comparisons on a per group basis. Having it set to 0 is the exodiff default for variables and then 1e-15 for timesteps (per the [Exodiff doco](https://sandialabs.github.io/seacas-docs/exo_util.pdf#page=11)). 

If floor is what you were referring to, this statement should be tweaked.